### ISSUE TITLE
File meaningful breadcrumbs

### ISSUE THREAD

**Issue by whit537** [gratipay/gratipay.com#4533]
Sentry has a breadcrumbs facility for tying together user actions across requests to construct a story for debugging context. Right now we get breadcrumbs all jumbled together as to be meaningless. Can we at least track by authenticated user if not by session?

**Comment by whit537** [comment]
Blog post announcement: https://blog.sentry.io/2016/05/04/breadcrumbs

Stubby generic doco: https://docs.sentry.io/learn/breadcrumbs/

Bit better Python doco: https://docs.sentry.io/clients/python/breadcrumbs/

Latter suggests that breadcrumbs are thread-scoped (not surprising coming from mitsuhiko; Flask depends heavily on thread-local context).

**Comment by whit537** [comment]
Checked w/ mitsuhiko & dcramer in IRC. Yes, they're thread-scoped. I confirmed that we're using gunicorn with the gthread worker and four threads (per `GUNICORN_OPTS` envvar at Heroku). dcramer suggests monkey-patching thread.locals.

Our breadcrumbs are pretty much complete noise right now.

![screen shot 2017-06-26 at 2 51 11 pm](https://user-images.githubusercontent.com/134455/27559865-4e1f3c66-5a7f-11e7-8840-9b64ada1d2dd.png)
